datalake/processing_stack.py

```diff
@@ -21,12 +22,32 @@
 class ProcessingStack(core.Stack):
     """Data Lake processing stack definition."""
 
-    # pylint: disable=too-many-locals
+    # pylint: disable=too-many-locals,too-complex
     def __init__(self, scope: core.Construct, stack_id: str, deploy_env, vpc, **kwargs) -> None:
         super().__init__(scope, stack_id, **kwargs)
 
+        # set resources removal policy for different environments
+        if deploy_env == "prod":
+            REMOVAL_POLICY = core.RemovalPolicy.RETAIN
```

Comment:
Local variables should be lower case.

Reply:
Should be fixed in 17923d87546f6713a687ded8ccb111eff35e4b34